presto/presto.sh

```diff
@@ -18,6 +18,8 @@ set -euxo pipefail
 # Use Python from /usr/bin instead of /opt/conda.
 export PATH=/usr/bin:$PATH
 
+readonly PRESTO_BASE_URL=https://repo1.maven.org/maven2/com/facebook/presto
```

Comment:
I'd suggest move this just before `readonly PRESTO_VERSION='

Reply:
Done. Good point, thanks!